II.pass-tests/08-get-average/get-average.test.js

```diff
@@ -1,4 +1,4 @@
-var average = require('./get-average');
+var {average} = require('./get-average');
```

Comment:
If you switch to using `module.exports = YOUR_EXPORT` in the implementation, you won't need to change this.

Reply:
ok